src/main/kotlin/com/colivery/serviceaping/security/SecurityConfiguration.kt

```diff
@@ -38,5 +56,4 @@ class SecurityConfiguration {
     fun reactiveAuthenticationManager(firebaseUserDetailsService: FirebaseUserDetailsService) =
```

Comment:
not exactly related to your changes, but maybe here it would be better to have the parameter type as "ReactiveUserDetailsService" instead of the specific Firebase sub-type... when we try to run with the profile "development" active, we need to be able to create a "DummyUserDetailsService" bean to be provided here